src/frontend/js/widgets/Dashboard/utils/dashboardRoutes.tsx

```diff
@@ -51,31 +44,15 @@ export function getDashboardRoutes() {
   return routes;
 }
 
-/**
- * Use `intl.formatMessage` to retrieve a path or label of a given route for the active locale.
- *
- * Currying function which takes first the type of attribute to translate (path or label)
- * then takes the intl object retrieved from the IntlProvider.
- * Finally, it takes a path and options to pass to `intl.formatMessage` method
- */
-
-enum RouteAttributes {
-  PATH = 'path',
-  LABEL = 'label',
-}
-
 const DASHBOARD_ROUTE_LABELS = {
   ...LEARNER_DASHBOARD_ROUTE_LABELS,
   ...TEACHER_DASHBOARD_ROUTE_LABELS,
 };
-const DASHBOARD_ROUTE_PATHS = {
-  ...LEARNER_DASHBOARD_ROUTE_PATHS,
-  ...TEACHER_DASHBOARD_ROUTE_PATHS,
-};
+
 type DashboardPath = LearnerDashboardPaths | TeacherDashboardPaths;
 
-const getDashboardRouteAttribute =
-  (attribute: RouteAttributes) =>
+/** Get the provided dashboard route label in the active locale */
+export const getDashboardRouteLabel =
```

Comment:
This function is just alias for `intl.formatMessage` and therefor not needed

Reply:
Same, encapsulation purpose, you're vote @jbpenrath ?

Reply:
In this case, I think I prefer the encapsulation as I think it's better for dx. This is simpler to understand to have to call `getDashboardRouteLabel` than to have to import `DASHBOARD_ROUTE_LABELS` objects then use into a `intl.formatMessage`